Helm lint gives me
```
[ERROR] templates/deployment.yaml: unable to parse YAML
	error converting YAML to JSON: yaml: line 30: found character that cannot start any token
```
However looking at [templates/deployment.yaml#30](templates/deployment.yaml#30) i cannot understand what is wrong...

--- a/stable/prometheus-postgres-exporter/templates/_helpers.tpl
+++ b/stable/prometheus-postgres-exporter/templates/_helpers.tpl
@@ -48,5 +48,5 @@ Create the name of the service account to use
 Set DATA_SOURCE_URI environment variable
 */}}
 {{- define "prometheus-postgres-exporter.data_source_uri" -}}
-{{ printf "%s:%s/%s?sslmode=%s" .Values.config.datasource.host .Values.config.datasource.port .Values.config.datasource.database .Values.config.datasource.sslmode }}
+{{ printf "%s:%s/%s?sslmode=%s" .Values.config.datasource.host .Values.config.datasource.port .Values.config.datasource.database .Values.config.datasource.sslmode | quote }}
 {{- end }}

